### PR TITLE
Polish translation updated

### PR DESCRIPTION
**Pull request by napcok**

### CODE REVIEW

File: src/translations/libfm-qt_pl.ts [hunk 1]
```diff
@@ -1118,132 +1118,132 @@ Zmodyfikowany: %3</translation>
...
         <location filename="../filesearch.ui" line="223"/>
         <source>Case insensiti&amp;ve</source>
-        <translation type="unfinished"></translation>
+        <translation>Nie uwzględniaj wielkości liter</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="230"/>
         <source>&amp;Use regular expression</source>
-        <translation type="unfinished"></translation>
+        <translation>Użyj wyrażeń regularnych</translation>
```
Comment (philippwiesemann) [inline src/translations/libfm-qt_pl.ts]: The accelerator is missing in translation.

File: src/translations/libfm-qt_pl.ts [hunk 2]
```diff
@@ -855,32 +855,32 @@ Czy zamiast tego usunąć te pliki?</translation>
...
         <location filename="../placesview.cpp" line="346"/>
         <source>Open in New Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>Otwórz w nowej k&amp;arcie</translation>
     </message>
     <message>
         <location filename="../placesview.cpp" line="349"/>
         <source>Open in New Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Otwórz w nowym o&amp;knie</translation>
```
Comment (f2404) [inline src/translations/libfm-qt_pl.ts]: Accelerator is not needed here.

File: src/translations/libfm-qt_pl.ts [hunk 3]
```diff
@@ -1118,132 +1118,132 @@ Zmodyfikowany: %3</translation>
...
         <location filename="../filesearch.ui" line="214"/>
         <source>File contains:</source>
-        <translation type="unfinished"></translation>
+        <translation>Plik zawiera:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="223"/>
         <source>Case insensiti&amp;ve</source>
-        <translation type="unfinished"></translation>
+        <translation>Nie uwzględniaj wielkości liter</translation>
```
Comment (f2404) [inline src/translations/libfm-qt_pl.ts]: Accelerator is missing here.

**Comment by PCMan** [comment]
Thank you. Please noted that the license of libfm-qt has been changed to LGPL.
If you're ok with that, I'll merge your PR. Thank you.

**Comment by napcok** [comment]
Of course, I'm ok with licence change. Thank you

**Comment by PCMan** [comment]
Merged. Thanks!

**Pull request closed without merging**